Clinical-Genomics/scout
# Update login page logos

This PR updates the logos on the login page and make the swedac logo configurable. To alter the logo you have to put a new logo in scout/server/blueprints/public/static and change the variable i `ACCREDITATION_BADGE` in `config.py`

![image](https://user-images.githubusercontent.com/12696913/126979284-c47f8286-13ca-43ea-9652-25a89ac1f768.png)

**To test**
1. Setup a scout demo instance
2. Verify that `ACCREDITATION_BADGE`
3. Verify that the swedac logo is being displayed on the login page
4. Set  `ACCREDITATION_BADGE` to null and restart the server
5. Verify that the logo is not displayed on the login page.

**Review:**
